cli/src/pcluster/validators/common.py

```diff
@@ -53,20 +53,21 @@ def __init__(self, raise_on_error=False):
     def _fail(self, message, level):
         raise ConfigValidationError
 
-    def _add_failure(self, message: str, level: FailureLevel, fields=()):
+    def _add_failure(self, message: str, level: FailureLevel, failed_params: List[Param] = None):
```

Comment:
`List[Param]=()` works, doesn't it? 
I am curious why you have to add another line of code for `if failed_params:`

Reply:
Fixed